src/it/java/org/jboss/tools/intellij/openshift/test/ui/OpenshiftNodeLoggedOutUITests.java

```diff
@@ -0,0 +1,335 @@
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        sharedSteps.restoreKubeConfig();
+    }
+
+    @AfterEach
+    public void afterEachCleanUp() {
+        try {
+            LOGGER.info("After test cleanup: Checking for any opened dialog window");
+            ContainerFixture dialogWindow = robot.find(ContainerFixture.class, byXpath("//div[@class='MyDialog']"));
+            dialogWindow.find(ComponentFixture.class, byXpath("//div[@class='JButton']"))
+                    .click();
+            currentClusterUrl = DEFAULT_CLUSTER_URL;
+        } catch (Exception e) {
+            LOGGER.info("After test cleanup: No dialog window opened");
+        }
+
+        try {
+            LOGGER.info("After test cleanup: Checking for any opened run window");
+            robot.find(ComponentFixture.class, byXpath("//div[@class='ToolWindowHeader'][.//div[@class='ContentTabLabel']]//div[@myaction.key='tool.window.hide.action.name']"))
+                    .click();
+        } catch (Exception e) {
+            LOGGER.info("After test cleanup: No run window opened");
+        }
+
+        try {
+            LOGGER.info("After test cleanup: Checking for opened Openshift view");
+            OpenshiftView view = robot.find(OpenshiftView.class);
+            robot.find(ComponentFixture.class, byXpath("//div[@class='BaseLabel']"), Duration.ofSeconds(2));
+            view.closeView();
+        } catch (Exception e) {
+            LOGGER.info("After test cleanup: Openshift view is not opened");
+        }
+    }
+
+    @Test
+    public void defaultNodeTest() {
+        logOut();
+        OpenshiftView view = robot.find(OpenshiftView.class);
+        view.openView();
+
+        waitFor(Duration.ofSeconds(20), () -> !view.getOpenshiftConnectorTree().findAllText(DEFAULT_CLUSTER_URL).isEmpty());
+        view.getOpenshiftConnectorTree().expand(DEFAULT_CLUSTER_URL);
+        view.waitForTreeItem("Please log in to the cluster", 60, 1);
+
+        assertFalse(view.getOpenshiftConnectorTree().findAllText("Please log in to the cluster").isEmpty());
+
+        view.closeView();
+    }
+
+    @Test
+    public void clusterLoginDialogTest() {
+        // Open the Openshift view
+        OpenshiftView view = robot.find(OpenshiftView.class);
+        sharedSteps.openLoginDialog(view);
+
+        sharedSteps.waitForComponentToAppear(20, 1, byXpath("//div[@class='MyDialog']"));
+        assertTrue(robot.findAll(ComponentFixture.class, byXpath("//div[@class='MyDialog']"))
+                .stream()
+                .anyMatch(ComponentFixture::isShowing));
+
+        // Close the dialog window
+        ContainerFixture dialogWindow = robot.find(ContainerFixture.class, byXpath("//div[@class='MyDialog']"));
+        dialogWindow.find(ComponentFixture.class, byXpath("//div[@text.key='button.cancel']"))
+                .click();
+        assertFalse(robot.findAll(ComponentFixture.class, byXpath("//div[@class='MyDialog']"))
```

Comment:
Instead of hardcoded paths in test it would be better to create a class that represents the object and call functions upon it